eng/vscode/README.md

```diff
@@ -88,8 +88,11 @@ Here are some cool prompts you can try across our supported Azure services:
 * "List containers in my Cosmos DB database"
```

Comment:
The diff on this is weird.  Make sure it is formatted correctly

Reply:
yes, it's weird, there is no format change, maybe this is a github comparation defect

Reply:
Reverted this and deferred for future action